exportCategories.php

```diff
@@ -69,7 +69,12 @@ protected function exportData(Mage_Catalog_Model_Category $category, $file, $dep
         echo '* ' . $category->getName() . sprintf(' (%s products)', $category->getProductCount()) . PHP_EOL;
         fputcsv($file, $data);
         if ($category->hasChildren()) {
-            $children = Mage::getModel('catalog/category')->getCategories($category->getId());
+            $children = Mage::getModel('catalog/category')
+                ->load($category->getId())
```

Comment:
Yup you are right, both can probably be deleted from there since they are not needed. The sort by position was to keep the order of the categories, but since the script exports/imports the actual category ID, I don't think there will be a problem with the categories order.